### ISSUE TITLE
[BITV] 9.3.3.2/3.1 - Comment section: The input fields for creating and editing comments lack visual labels. (2)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41884]
![image](https://github.com/nextcloud/server/assets/6078378/75417371-ab19-4bd2-8deb-4cf1dae5585e)

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-9f92015c98-v3-n1

9.3.3.2/11.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-9f92015c98-v11-n1

**Comment by susnux** [comment]
> But what do you mean by other activity? Is it about activities listed in this tab? If that is, should it be consistent? A form is not an activity.

Yes consider clicking edit on the comment, the expected behavior would be to change the comment to the editable version while keeping all other elements.

> [...] direct usage with a template ref.

That depends on what you are doing with it using the ref directly is pretty dangerous anyways as you then depend on the implementation details. Meaning if we want any methods to be exposed we should use `expose` and document them. For example if you need to access the inner contenteditable we could document and expose the ref to `contenteditable` of the component.

**Comment by JuliaKirschenheuter** [comment]
@susnux can this issue be moved to "Parked" (needs new BC Vue release)?

**Comment by susnux** [comment]
Waiting for release of https://github.com/nextcloud-libraries/nextcloud-vue/pull/4907
